client/src/test/java/org/spine3/client/ClientUtilShould.java

```diff
@@ -41,7 +42,7 @@ public void have_private_constructor() {
     public void generate_command_Ids() {
         final CommandId result = generateId();
 
-        assertFalse(result.getUuid().isEmpty());
+        assertFalse(idToString(result).isEmpty());
```

Comment:
We have to check the `UUID` field explicitly because if `generateId()` returns `null` this test will pass (the string will be `NULL`).